### ISSUE TITLE
lag issues

### ISSUE THREAD

**Issue by Latravant** [atperry7/pup_gearswap#17]
i just started using my pup for the first time in a year and when i loaded the pup.lua my game started lagging. i noticed this happened with other luas i use for my brd too. any clue what could be causing it?

**Comment by atperry7** [comment]
This is the first report I have heard of it causing lag. No one has reported in our discord either. 

So, it will be hard to say what might be causing it. 

If you disable all other addons and only run gearswap and pup.lua, does the lag still occur?

**Comment by Latravant** [comment]
ok just unloaded everything, changed to pup, reloaded gearswap and bam instant lag spikes

**Comment by Latravant** [comment]
unloaded gearswap and the spikes stopped

**Comment by Latravant** [comment]
i posted in the windower discord for help in case it has something to do with gearswap itself instead of the pup lua. but i was just thinking, the pup lua and the brd lua i mentioned both have somehting in common, they both have on screen menus, so thought maybe this could be causing the spikes somehow?

**Comment by Gojito** [comment]
+1 to the lag issue
It is not super noticeable on a single windower instance, but if dual boxing or more the lag on the character using the pup lua will be severe.

It only happens when on pup. With other jobs I can have 6 windower instances loaded without problems. With the pup lua, it starts to get unstable at around 3 instances, but the lag spikes are present even with 1 instance.

**Comment by Latravant** [comment]
Yeah I tried pup for the first time in awhile last week and it was horrible. I unloaded gear swap and the lag stopped

**Comment by atperry7** [comment]
Given the age of this issue, we will close for now and see if any new reports surface.
